discover/portal/articles/210-setting-up/01-system-settings/01-system-settings.markdown

```diff
@@ -0,0 +1,121 @@
+sometimes even code chunks, are configurable at several scopes, as discussed in
+the introductory article for this section. 
+
+To make configuration changes to an application in a system-wide fashion, use
+the System Settings application at Control Panel &rarr; Configuration &rarr;
+System Settings.
+
+![Figure 1: System Settings are accessed through the Control Panel.](../../../images/system-settings-categories.png)
+
+## Editing the Default Configuration [](id=editing-the-default-configuration)
+
+In System Settings, the configuration options are organized into logical
+sections (for example, Content) and categories (for example, Workflow) based on
+their functionality. There's also a Search bar to make finding configuration
+entries easier. 
+
+![Figure 2: System Settings are organized by component.](../../../images/system-settings-nav-search.png)
+
+Changing an application's default configuration options in System Settings isn't
+difficult. Once you find what you're looking for, simply click the name of the
+component you want to configure, or click the actions button
+(![Actions](../../../images/icon-actions.png)) then click *Edit*. Make any
+changes you'd like, then click *Save*. Your configuration changes are saved and
+applied throughout the system. 
+
+![Figure 3: After saving changes to a configuration, the actions *Reset Default Values* and *Export* are available for it.](../../../images/system-settings-actions.png)
+
+Keep in mind that some configurations are available at more than one scope. Once a
+configuration change is made at a more granular scope, making a change at the
+system level won't do anything. 
+
+For example, whether to allow comments on Blogs Entries is configurable. Set the default behavior is set at Control Panel &rarr; Configuration &rarr; System Settings &rarr; Blogs. In the Blogs entry under Widget Scope, disable the *Enable Comments* checkbox. 
+
+Now add a Blog Entry to a site's Content &rarr; Blogs application. Then go to a
+public page and add the Blogs Widget to the page. Click the Actions button
+(![Actions](../../../images/icon-actions.png)) for the Widget and select
+*Configuration*. You'll see the same Enable Comments checkbox, and its default
+is now false (unchecked). Checking the box in the Widget Configuration screen
+breaks its link with the System Settings entry. Changing the System
+Settings configuration does nothing to this Widget Instance anymore.
+
+If you make some regrettable configuration decisions and can't recall exactly
+what you did, start over by clicking the actions button
+(![Actions](../../../images/icon-actions.png)), then clicking *Reset Default
+Values*. 
+
+## Configuration Scope [](id=configuration-scope)
+
+While browsing the categories of System Settings, you'll notice that clicking
+into a topic (for example, Blogs) reveals entries at different scopes.each entry
+has a *Name* and a *Scope*. This scope is different than the overall
```

Comment:
I wouldn't say that it's different, it's the exact same scope.

What the menu is doing is informing the scope of the configuration. What the System Settings allows in each case is to set the configuration at the system scope, which will act as the default configuration for all scopes other than system scope itself.